SleepEngine/Engine/Config/ILoadable.h

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Taras Martyniuk
+#pragma once
+
+class Loadable

```

Comment:
So you've named the file ILoadable but class Loadable?

Reply:
forgot to change